tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py

```diff
@@ -7,11 +7,17 @@
 
 
 class StatusBar(Base):
```

Comment:
The status bar is a region of the system app. With it's own functionality and objects.

Not sure I see the advantage of making this a `PageRegion`